README.rst

```diff
@@ -7,21 +7,21 @@ pip-tools = pip-compile + pip-sync
 A set of command line tools to help you keep your ``pip``-based packages fresh,
 even when you've pinned them.  You do pin them, right? (In building your Python application and its dependencies for production, you want to make sure that your builds are predictable and deterministic.)
 
-.. image:: https://github.com/jazzband/pip-tools/raw/master/img/pip-tools-overview.svg
+.. image:: https://github.com/jazzband/pip-tools/raw/main/img/pip-tools-overview.svg
```

Comment:
The old link is broken (404) and breaks the linkcheck-docs workflow.  See eg. https://github.com/jazzband/pip-tools/actions/runs/3514152156/jobs/5887815325